arch/arm/soc/atmel_sam0/samd/dts.fixup

```diff
@@ -0,0 +1,6 @@
+/* SoC level DTS fixup file */
+
+#define FLASH_DEV_BASE_ADDRESS		ATMEL_SAM0_NVM_CTRL_41004000_BASE_ADDRESS_0
```

Comment:
nit: Atmel call it the NVMCTRL unit, so s/NVM_CTRL/NVMCTRL/ everywhere.

Reply:
Fixed.